package.json

```diff
@@ -46,7 +46,7 @@
   ],
   "scripts": {
     "prepublishOnly": "npm run build",
-    "build": "tsup src/index.ts --format esm,cjs --dts --clean",
+    "build": "tsup src/index.ts --format esm,cjs --dts --clean; node -e 'require(`node:fs/promises`).copyFile(`dist/index.d.ts`, `dist/index.d.cts`)'",
```

Comment:
Actually, this change might no longer be needed at all. 5 days ago a new version of `tsup` was released and it already handles this problem for you:
https://github.com/egoist/tsup/releases/tag/v7.1.0

Reply:
Thanks for notifying me of that development. I already updated tsup, but hadn't checked the output.